src/olympia/addons/indexers.py

```diff
@@ -391,6 +391,7 @@ def get_mapping(cls):
                         'status': {'type': 'byte'},
                         'permissions': {'type': 'keyword', 'index': False},
                         'optional_permissions': {'type': 'keyword', 'index': False},
+                        'host_permissions': {'type': 'keyword', 'index': False},
```

Comment:
Not strictly needed since we aren't exposing that in the search API yet, but it can't hurt to start recording it in ES I guess... Will require a reindex.